Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -384,6 +419,14 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
             break;
 
         case WI.InlineSwatch.Type.Variable: {
+            if (this._variableType) {
```

Comment:
can we be more specific like `if (this._variableType === WI.InlineSwatch.Type.Color) {` so that it's more future-proof?